File: src/sdk-reference/cpp/1/auth/delete-my-credentials/snippets/delete-my-credentials.cpp
try {
  kuzzle->auth->login("local", R"({"username":"foo","password":"bar"})");
  kuzzle->auth->deleteMyCredentials("local");

  std::cout << "Credentials Successfully deleted" << std::endl;
} catch (kuzzleio::KuzzleException &e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/csharp/1/auth/delete-my-credentials/snippets/delete-my-credentials.cs
try {
  kuzzle.auth.login("local", @"{""username"":""foo"",""password"":""bar""}");
  kuzzle.auth.deleteMyCredentials("local");

  Console.WriteLine("Credentials Successfully deleted");
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.Message());
}
